engine/api/src/main/scala/pl/touk/nussknacker/engine/api/Context.scala

```diff
@@ -8,6 +8,8 @@ object Context {
 
 }
 
+class ContextId(val id: String) extends AnyRef
```

Comment:
`extends AnyRef` doesn't give anything. Do you mean `extends AnyVal`? Scala value classes have some problems, but I don't remember what exactly. Also IMO would be nice to make this serializable and comparable in case of storing it in some map. Regards to that maybe would be better to use `case class`?

Reply:
Thanks. Yes it was supposed to be a value class (`extends AnyVal`). Anyway, I've replaced it with case class.